frontend/pages/hosts/details/HostDetailsPage/modals/SelectReportModal/_styles.scss

```
@@ -1,19 +1,18 @@
+.select-report-modal {
+  max-height: 90%;
   overflow: hidden;
 
   .modal__content-wrapper {
-    height: 95%;
+    max-height: 95%;
   }
 
   .modal__content {
     @include vertical-modal-layout;
     height: 100%;
```

Stylelint (17.10.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)